src/soundio/sounddevicenetwork.cpp

```diff
@@ -89,15 +90,13 @@ SoundDeviceError SoundDeviceNetwork::open(bool isClkRefDevice, int syncBuffers)
         // Update the samplerate and latency ControlObjects, which allow the
         // waveform view to properly correct for the latency.
         ControlObject::set(ConfigKey("[Master]", "latency"),
-                m_audioBufferTime.toDoubleMillis());
+                requestedBufferTime.toDoubleMillis());
         ControlObject::set(ConfigKey("[Master]", "samplerate"), m_dSampleRate);
-        ControlObject::set(ConfigKey("[Master]", "audio_buffer_size"),
-                m_audioBufferTime.toDoubleMillis());
```

Comment:
I guess it's highly unlikely that this CO is used somewhere in mappings or skins - and therefore can be removed.

Reply:
I wanted to mention that it also needs to be removed from the Mixxx Controls section in the manual, but that CO does not seem to be documented anyway. 🤷